iOS/WAStickersThirdParty/StickerPackViewController.swift

```diff
@@ -93,6 +93,8 @@ class StickerPackViewController: UIViewController, UICollectionViewDataSource, U
         let buttonLandscapeSize: CGSize = CGSize(width: 250.0, height: 50.0)
         let buttonBottomMargin: CGFloat = 20.0
 
+        guard let view = view else { return }
```

Comment:
Why is this needed?

Reply:
Because `view`'s type is implicitly-unwrapped optional UIView. 
If we don't unwrap the view property warnings appear in the following lines.


![Screen Shot 2020-01-28 at 10 51 19](https://user-images.githubusercontent.com/19268363/73245197-bc7a2500-41bc-11ea-8ddc-0f803f643c8d.png)